Mathlib/Algebra/EuclideanDomain/Instances.lean

```diff
@@ -35,7 +35,7 @@ instance Int.euclideanDomain : EuclideanDomain ℤ :=
       not_lt_of_ge <| by
         rw [← mul_one a.natAbs, Int.natAbs_mul]
         rw [←Int.natAbs_pos] at b0
-        exact Nat.mul_le_mul_of_nonneg_left b0 }
```

Comment:
FWIW, this old (and incorrect) name is left from https://github.com/leanprover-community/lean/pull/366

Reply:
At least the spurious assumption was eliminated!